cli/eac.js

```diff
@@ -200,6 +210,11 @@ const main = async (_) => {
     scanner.start(program.milliseconds, conf)
     setTimeout(() => Repl.start(conf, program.milliseconds), 1200)
 
+    if (analytics) {
+      const addresses = conf.wallet.getAddresses()
+      analytics.startAnalytics(addresses[0]);
```

Comment:
It's possible for someone to be running multiple accounts from the same client. Do we count this as one timenode or multiple? Suppose it doesn't matter if we're just getting the rough estimate of number of timenodes.

Reply:
I guess, we just need the number of instances running. Which is basically, one, per running cli, as the multiple addresses run from the same instance